### PR TITLE
Feature/dvr low latency

### PR DESCRIPTION
**Pull request by SecondeJK**
Changes to allow DVR and Low Latency HLS features when starting a Broadcast

## Description
Changes to allow DVR and Low Latency HLS features when starting a Broadcast

## Motivation and Context
In line with the API spec for DVR and lowLatency modes, this PR is due for release to continue evolving the SDK in line with Vonage Video products.

## How Has This Been Tested?
New tests added with validators

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/OpenTok/Broadcast.php [hunk 1]
```diff
@@ -114,6 +114,22 @@ public function __get($name)
                 return null;
         }
     }
+
+	public function isHls(): bool
```
Comment (matt-lethargic) [inline src/OpenTok/Broadcast.php:118]: Is it a good idea to return these properties as properties of the main broadcast object?

**Review by matt-lethargic — CHANGES_REQUESTED** [review]

File: src/OpenTok/Broadcast.php (hunk 1 above)
Comment (SecondeJK) [reply src/OpenTok/Broadcast.php:118]: Methods wrapping around properties like this is pretty common in PHP

File: src/OpenTok/Util/Validators.php [hunk 2]
```diff
@@ -312,6 +312,18 @@ public static function validateBroadcastId($broadcastId)
             );
         }
     }
+
+	public static function validateBroadcastOutputOptions(array $outputOptions)
+	{
+		if (
+			isset($outputOptions['lowLatency'], $outputOptions['dvr'])
+			&& $outputOptions['lowLatency'] === true && $outputOptions['dvr']
```
[the remark this answers is not in the document]
Comment (SecondeJK) [reply src/OpenTok/Util/Validators.php:320]: aw man, good catch, it most certainly does

[the remark this answers is not in the document]
Comment (SecondeJK) [reply src/OpenTok/Util/Validators.php:320]: which means my test has a false positive.

**Review by jeffswartz — CHANGES_REQUESTED** [review]
Please rev the version to 4.11.0 (in Util/Client.php and elsewhere). 

Also, maybe this should target the dev branch and we can add this PR to the release (if it looks good): https://github.com/opentok/OpenTok-PHP-SDK/pull/297.

File: src/OpenTok/Broadcast.php [hunk 3]
```diff
@@ -114,6 +114,22 @@ public function __get($name)
...
+
+	public function isLowLatency(): bool
+	{
+		return $this->data['settings']['hls']['lowLatency'];
+	}
+
+	public function isDvr(): bool
+	{
+		return $this->data['settings']['hls']['dvr'];
+	}
```
Comment (jeffswartz) [inline src/OpenTok/Broadcast.php:131]: Why are these implemented as functions? Other properties of the Broadcast are implemented as properties (succh as `isStopped`, `streamMode`, etc.

**Review by jeffswartz — CHANGES_REQUESTED** [review]

File: src/OpenTok/Broadcast.php (hunk 3 above)
Comment (jeffswartz) [reply src/OpenTok/Broadcast.php:131]: Also, can please add docs (like we do for `@property boolean $isStopped`).

Comment (SecondeJK) [reply src/OpenTok/Broadcast.php]: done

**Review by jeffswartz — CHANGES_REQUESTED** [review]

File: src/OpenTok/OpenTok.php [hunk 4]
```diff
@@ -637,6 +637,26 @@ public function disableForceMute(string $sessionId, array $options): bool
      *    <a href="https://tokbox.com/developer/guides/archive-broadcast-layout/#stream-prioritization-rules">stream
      *    prioritization rules</a>.</li>
      *
+     *    <li><code>outputs</code> (Array) &mdash;
```
[the remark this answers is not in the document]
Comment (SecondeJK) [reply src/OpenTok/OpenTok.php:640]: I've added these into the docs, and have a new validator for this

**Review by jeffswartz — APPROVED** [review]

**Pull request merged**